unit-tests/src/test/scala/scala/scalanative/regex/Utils.scala

```diff
@@ -0,0 +1,210 @@
+// Copyright 2010 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package scala.scalanative
+package regex
+
+// Various constants and helper utilities.
+object Utils {
```

Comment:
Hum, where does this come from? Is it new? Is it needed here?

Reply:
I removed this initially but it was used in the `ExecTest.disabled` test. `ExecTest` has been ported and `@Ignore`ed because we see compile errors and it runs too slow. It runs despite the compile errors. I put some documentation in the file and commented out the majority of the test so it wouldn't cause errors.

`ParserTest` is also very slow so we probably need an issue to work on the regex stuff. 

Everything is about the best I can do with `regex` package. @LeeTibbert can probably add a ticket to address future work needed as I think he did most of the work and has the best knowledge on what needs to be done going forward.

Reply:
Should be good to go now.